src/Raven.Client/Server/DatabaseRecord.cs

```diff
@@ -50,6 +53,8 @@ public DatabaseRecord(string databaseName)
 
         public PeriodicBackupConfiguration PeriodicBackup { get; set; }
 
+        public Dictionary<string, SubscriptionRaftState> Subscriptions;
```

Comment:
Why is the key a `string`? Since we are using subscription ids to identify it, it would be better to use a `long`